pkg/controller/pxc/controller.go

```diff
@@ -288,6 +288,22 @@ func (r *ReconcilePerconaXtraDBCluster) Reconcile(request reconcile.Request) (re
 		return reconcile.Result{}, err
 	}
 
+	for _, pxcService := range []corev1.Service{*pxc.NewServicePXC(o), *pxc.NewServicePXCUnready(o)} {
```

Comment:
should we really query this objects?
We need only Name and namespace from them, which is simply
Namespace: o.Namespace
Name: o.Name+"-"+appName